docs/Tutorials/Privacy/Configuring-Multi-Tenancy.md

```diff
@@ -0,0 +1,112 @@
+description: Multi-tenancy
+<!--- END of page meta data -->
+
+# Configuring a Multi-tenant Node
+
+### Prerequisites
+
+* [Privacy-enabled network](Configuring-Privacy.md)
+
+Each Besu and associated Orion node in a privacy-enabled network can be
+configured to host [multiple tenants](../../Concepts/Privacy/Multi-Tenancy.md).
+
+In this tutorial we will add tenants to the `Node-1` Besu and Orion node in a 
+[privacy-enabled network](Configuring-Privacy.md). 
+
+```bash
+IBFT-Network/
+├── Node-1
+│   ├── data
+│   ├── Orion
+├── Node-2
+│   ├── data
+│   ├── Orion
+└── Node-3
+    ├── data
+    ├── Orion 
+```
+
+!!! tip
```

Comment:
note?

Reply:
I was torn between "note" or "tip". Will update to "note".